### PR TITLE
[MistySOM/rzg2l]  Merge feature_hwtest branch (g2l & v2l patches)

### PR DESCRIPTION
**Pull request by gfinlay**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "Build/meta-mistysom"]
 	path = Build/meta-mistysom
-	url = https://github.com/MistySOM/meta-mistysom.git
+	branch = mistysom-io
```
Comment (mistyron) [inline .gitmodules:3]: this should not go into `develop`, should it?

**Review by mistyron — CHANGES_REQUESTED** [review]
why is the branch of the meta-mistysom submodule updated, to mistysom-io?That  doesn't look right, does it?

**Review by mistyron — COMMENTED** [review]
@matinlotfali is working on moving all kernel config patches into meta-mistysom to simplify management as they apply to  G2L & V2L kernels.

**Review by mistyron — APPROVED** [review]
sync with @matinlotfali to get these fragments copied into meta-mistysom, please

**Comment by gfinlay** [comment]
> sync with @matinlotfali to get these fragments copied into meta-mistysom, please

Done

**Pull request merged**